packages/node_modules/@node-red/editor-client/locales/de/README.md

```diff
@@ -0,0 +1,36 @@
+This folder contains the german translations.
```

Comment:
What does this readme say? What "rules" are you introducing?

I'm happy for us to document an i18n process, but it should be consistent for all languages and shouldn't be documented in a file that ends up being distributed with the node-red npm module.

Reply:
This are german-specific rules, how to translate englisch to german, but without change of the content or meaning.
For example, some englisch words are 1:1 overtaken in german, but other words are translated, and this without a general rule which is the right way.
An other standing discussion is the translation of "the". In german we have the 3 articles "der", "die" and "das" and for unusual nouns we have to clarify/dicide/unify it.
To get a consistent german doc, I started to create a common dictionary with englisch and translated german words.

Reply:
@knolleary 
I just added the English translation to https://github.com/heikokue/node-red/blob/i18n-de/manifest_dictionary/packages/node_modules/@node-red/editor-client/locales/de/README.md

Reply:
As I said, I'm happy to have documentation for this, but it should not be added as a readme to this directory. Please remove it from this PR and we can discuss where it should go.

Reply:
yes I agree, it's not the best location, because it's an global topic. I already asked the question where to store in the forum, but up to now it got no answer. What is the best folder to store it and the german dictionary?